package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "@microsoft/atlas-design",
+	"version": "1.0.0",
+	"description": "The Atlas Design System used by Microsoft's Developer Relations platform.",
```

Comment:
```
"private": true
```
to ensure tooling never publishes the monorepo package
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private

Nice to have homepage as well https://docs.npmjs.com/cli/v7/configuring-npm/package-json#homepage

Reply:
updated